include/circt/Dialect/LLHD/IR/LLHDTypesImpl.td

```diff
@@ -17,26 +17,6 @@ class LLHDType<string name> : TypeDef<LLHD_Dialect, name> { }
 // Type declarations
 //===----------------------------------------------------------------------===//
 
-// Declares the llhd::SigType in C++.
-def SigTypeImpl : LLHDType<"Sig"> {
-  let summary = "signal type";
-  let description = [{
-    Signals correspond directly to wires in a physical design, and are used to
-    model propagation delays and timing. Signals are used to carry values across
-    time steps in the LLHD execution model.
-  }];
-
-  let mnemonic = "sig";
-  let parameters = (ins "::mlir::Type":$underlyingType);
-  let assemblyFormat = "`<` $underlyingType `>`";
-
-  let builders = [
-    AttrBuilderWithInferredContext<(ins "::mlir::Type":$underlyingType), [{
-      return $_get(underlyingType.getContext(), underlyingType);
-    }]>
-  ];
-}
-
 // Declares the llhd::PtrType in C++.
 def PtrTypeImpl : LLHDType<"Ptr"> {
```

Comment:
Do you think we might be able to use `memref`s as a replacement for `!llhd.ptr` in the future?

Reply:
In theory yes, but we might not want to pull in all the complexity of `memref` in practice.